tests/dec/test_dec.py

```diff
@@ -411,6 +411,13 @@ def test_master_DECAYdotDEC_file():
 
     assert p.number_of_decays == 506
 
+def test_BELLE2_decfile():
+    p = DecFileParser(DIR / '../../decaylanguage/data/DECAY_BELLE2.DEC')
+    p.parse()
+
+    # Just check the dec file will parse since I do not know 
+    # how many decays are in the dec file.
+    assert p.number_of_decays > 0
```

Comment:
You do know precisely how many decays are in the file, once you parse successfully. I reckon it is fine to just check for that number in the test, knowing that the test is likely to need an update if the dec file gets updated.

Reply:
Oh I see, it is a guard for future changes. Thanks for pointing this out.